### PR TITLE
Add 100 smooth chat UX

### PR DESCRIPTION
**Pull request by oskarth**
cc @ricardosaavedra (see you are not on Status GH yet :o - asked for permissions)

### CODE REVIEW

File: ideas/100-smooth-chat-ux.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
+<!-- Please Review https://docs.google.com/document/d/1CaFM2ZXGOKf05_LXMPJeNNy5qJOdAq91EF2Gn2QUBFI/edit# for more details -->
```
Comment (oskarth) [inline ideas/100-smooth-chat-ux.md]: Can remove this stuff

File: ideas/100-smooth-chat-ux.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+<!-- in PR the document should be named as`DEV#1-title.md` -->
+
+## Preamble
+
+    Idea: UX 100
+    Title: Smooth chat UX
+    Status: Draft
+    Created: date created on 2018-03-23
```
Comment (oskarth) [inline ideas/100-smooth-chat-ux.md]: minor: format (2018-03-23)

File: ideas/100-smooth-chat-ux.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
+<!-- Please Review https://docs.google.com/document/d/1CaFM2ZXGOKf05_LXMPJeNNy5qJOdAq91EF2Gn2QUBFI/edit# for more details -->
+<!-- in PR the document should be named as`DEV#1-title.md` -->
+
+## Preamble
+
+    Idea: UX 100
```
Comment (oskarth) [inline ideas/100-smooth-chat-ux.md]: 100-smooth-chat-ux maybe?

File: ideas/100-smooth-chat-ux.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+<!-- Each contributor pledges to the idea with their FOCUS value. (hours per week) -->
+<!-- Here all roles in swarm are defined and filled, one of the contributors should responsibility of the Idea as Lead. -->
+
+<!-- Testing/Evaluation support role is also mandatory to check in on specified Goal dates or earlier. -->
+
+<!-- Lead Contributor is the Owner of the Idea. If required, they can get support from a PM, but should be responsible for end to end execution of the Idea. This includes ensuring appropriate resources are allocated, setting realistic timelines and milestones, and any post-launch metrics or bug fixes that are attributed to the Idea -->
+<!-- A swarm requires at minimum 3 contributors and 1 evaluator/tester -->
+<!-- 'Contributor' should be replaced with a descriptive role type. -->
+- Lead Contributor: @ricardosaavedra
+- Testing & Evaluation: <!-- @username -->
```
Comment (oskarth) [inline ideas/100-smooth-chat-ux.md]: What does the interest look like from other core contributors (Clojure devs, UX(R), PM) so far?

File: ideas/100-smooth-chat-ux.md [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+<!-- Testing/Evaluation role accepts responsbility to checkin at Goal dates, -->
+<!-- forces discussion to continue implementation or recommend disband and post-mortem. -->
+Every beginning of a new cycle (you can call it sprint if you'd like), we will decide together what to focus on for the upcoming two weeks as well as discussing current problems, technical challenges and the best solutions (trade-off between implementation cost and UX).
+As ideas and design solutions start to emerge, we will get together to validate/critique the proposed solutions until we arrive at a group consensus.
+
+
+### Product Description
+<!-- What functionality are you adding? What will this look like from a user perspective? Why is this important? -->
+
+**V1 - quick fixes**
```
Comment (oskarth) [inline ideas/100-smooth-chat-ux.md]: Good stuff

File: ideas/100-smooth-chat-ux.md [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+- Chat view / minior UI fixes 
+- Chat view / message space optimization
+- Chat view / timestamps
+- Chat view / adding/blocking new contact flow
+- Keyboard / use cases behaviors
+- Keyboard / maximum message width
+- Keyboard / fixing console position
+- Offline states and error handling 
+
+**V2 - group chat improvements**
```
Comment (oskarth) [inline ideas/100-smooth-chat-ux.md]: Possibly leave this out of scope if we don't need group chat for MVP, i.e. might make more sense to have V3 before V2? Don't know - @chadyj might have an opinion

File: ideas/100-smooth-chat-ux.md [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+- Home screen / search function 
+- Home screen / loading messages experience
+
+For more details about each one of those fixes, please visit: https://docs.google.com/document/d/1AyU9mWMG1MxNPPBr1kidIl7eMJzPhbUjJjUNBBxZpg8/edit
+
+### Team & Dependencies
+<!-- Are there bugs or feature requests in other repositories that are part of this Idea? -->
+<!-- There is no approval unless the idea requires to be reviewed by supporting organelles (Financial, Hiring, or Design). -->
+<!-- The Swarm must develop a fully fleshed out Requirements document for the idea to proceed, to the satisfaction of participants. -->
+Ideally, we would have at least 2 developers (Whisper, Clojure) in the team acting as a stakeholder in the process.
```
Comment (oskarth) [inline ideas/100-smooth-chat-ux.md]: What do you mean by Whisper developer? What knowledge is required? I believe Clojure devs hopefully have enough knowledge on application protocol layer for this.

What does the overlap with this and 99 look like btw?

File: ideas/100-smooth-chat-ux.md [hunk 8]
```diff
@@ -0,0 +1,119 @@
...
+
+Even though you won't be implementing those designs directly, you should act as if you were, so that what comes out of this process isn't a wishfull thinking of our collective imagination; bearing no relationship with reality (time constraints and limited resources). 
+
+<!--### Minimum Viable Product-->
```
Comment (oskarth) [inline ideas/100-smooth-chat-ux.md]: Would be great to define this before getting started - is this V1?

File: ideas/100-smooth-chat-ux.md [hunk 9]
```diff
@@ -0,0 +1,119 @@
...
+
+Testing Days required: <!-- Days required at the end of development for testing -->
+
+## Success Metrics
+
+- average of 10 messages a day by each active user 
+- messaging experience feels good: 90% of surveyed users feel like the messaging experience is solid enough compared to new apps
+- 80% of users that installed the app used the messaging feature in their first week (simple and inviting)
+
+
```
Comment (oskarth) [inline ideas/100-smooth-chat-ux.md]: What's the exit criteria for this swarm?

File: ideas/100-smooth-chat-ux.md [hunk 10]
```diff
@@ -0,0 +1,119 @@
...
+## Dates
+Goal Date:  <!-- Date for evaluation in ISO 8601 (yyyy-mm-dd) format --> 
+
+Description: <!-- Description of Deliverables-->
+
+Testing Days required: <!-- Days required at the end of development for testing -->
+
+## Success Metrics
+
+- average of 10 messages a day by each active user
```
Comment (oskarth) [inline ideas/100-smooth-chat-ux.md]: Do we have baseline numbers for these states? If not, is this part of the initial scope perhaps? Would be useful from UXR POV, and might be some overlap with 68-core-metrics and 99-confidence

**Comment by martinklepsch** [comment]
Might be worth using a directory for this one as there are many relevant images in the issue: #100

**Comment by oskarth** [comment]
Closing as nothing has changed in PR last week and Ricardo left. If someone wants to pick it up there's bunch of work done here: https://github.com/status-im/ideas/issues/100

**Pull request closed without merging**